core/templates/services/contextual/window-ref.service.ts

```diff
@@ -22,11 +22,15 @@ import { Injectable } from '@angular/core';
   providedIn: 'root'
 })
 export class WindowRef {
+  /**
+  * @returns the global native browser window object
```

Comment:
@larakhdavies The comments should start with capital letters and end with a period(.) Can you do this here and below as well?
```suggestion
  * @returns The global native browser window object.
```